javalib/src/main/scala/java/nio/channels/SelectorImpl.scala

```diff
@@ -0,0 +1,158 @@
+package java.nio.channels
+
+import java.io.IOException
+import java.net.Net
+import java.nio.channels.spi.{
+  AbstractSelectableChannel,
+  AbstractSelectionKey,
+  AbstractSelector,
+  SelectorProvider
+}
+import java.util
+import scala.collection.mutable
```

Comment:
This comment is probably relevant other places in the 41 files.

There has been a drive to use Java methods in javalib and not
use Scala Library methods. This is especially true for new code.
The concern has to do with circularity javalib being implemented
in terms of scala library (no concern about core language) and 
the latter being implemented in terms of javalib.  Now that _could_
be done without link time circularity and we have gotten away with
it to date, but each instance introduced of javalib using Scala standard
library increases the chance.  



Because of historical rapid bootstrapping, when javalib was even more
incomplete, you will see instances of "scala.mumble.mutable" scattered
across existing code. 

In the past week, I stumbled across one such
that was introduced in the past year.

Someday we hope to remove the automatic inclusion of Predef and
track down the remaining Scala Library usages.   Until we do that,
we will never have any peace and must rely upon Review.

By having used it many times, I usually use javalib `HashMap` and `HashSet` instead of the corresponding unqualified `scala.mutable.Map`.  Each instance must be examined because 
in some cases `TreeMap` or `LinkedHashMap` is more suitable.